### PR TITLE
bugfix: support exists in where condition

### LINKED ISSUE #1737: MySQL can not support exists in where condition
- [ ] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description
This is the sql
```
UPDATE a_ru_temp tmp
SET tmp.cur_node_ins_id = (
	SELECT b.node_ins_id
	FROM b_ru_temp b
	WHERE tmp.cur_process_ins_id = b.process_ins_id
		AND tmp.cur_node_id = b.NODE_ID
)
WHERE EXISTS (
		SELECT 1
		FROM b_ru_temp b
		WHERE tmp.cur_process_ins_id = b.process_ins_id
			AND tmp.cur_node_id = b.NODE_ID
	)
```

### Ⅱ. Describe what happened

  If there is an exception, please attach the exception trace:

```
[ERROR] 2019-10-10 10:48:48 [http-nio-8012-exec-8] io.seata.rm.datasource.exec.AbstractDMLBaseExecutor (AbstractDMLBaseExecutor.java:104) - exception occur
java.lang.IllegalArgumentException: unexpected WHERE expr: SQLExistsExpr
	at io.seata.rm.datasource.sql.druid.MySQLUpdateRecognizer.getWhereCondition(MySQLUpdateRecognizer.java:117)
	at io.seata.rm.datasource.exec.BaseTransactionalExecutor.buildWhereCondition(BaseTransactionalExecutor.java:134)
	at io.seata.rm.datasource.exec.UpdateExecutor.buildBeforeImageSQL(UpdateExecutor.java:74)
	at io.seata.rm.datasource.exec.UpdateExecutor.beforeImage(UpdateExecutor.java:62)
	at io.seata.rm.datasource.exec.AbstractDMLBaseExecutor.executeAutoCommitFalse(AbstractDMLBaseExecutor.java:71)
	at io.seata.rm.datasource.exec.AbstractDMLBaseExecutor.executeAutoCommitTrue(AbstractDMLBaseExecutor.java:93)
	at io.seata.rm.datasource.exec.AbstractDMLBaseExecutor.doExecute(AbstractDMLBaseExecutor.java:57)
	at io.seata.rm.datasource.exec.BaseTransactionalExecutor.execute(BaseTransactionalExecutor.java:96)
	at io.seata.rm.datasource.exec.ExecuteTemplate.execute(ExecuteTemplate.java:101)
	at io.seata.rm.datasource.exec.ExecuteTemplate.execute(ExecuteTemplate.java:47)
	at io.seata.rm.datasource.PreparedStatementProxy.executeUpdate(PreparedStatementProxy.java:83)
```

### Ⅲ. Describe what you expected to happen
I just want to know why SQLExistsExpr is not support, and is there a plan to support this issue?

image

### LINKED ISSUE #1870: log.error not print detail log
- [ ] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description
image

### Ⅱ. Describe what happened

  If there is an exception, please attach the exception trace:

```
Just paste your stack trace here!
```

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. xxx
2. xxx
3. xxx

### Ⅵ. Environment:

- JDK version :
- OS :
- Others:

### PR DESCRIPTION
**Pull request by yuhuangbin**
### Ⅰ. Describe what this PR did
bugfix: support exists in where condition

### ⅠI. Does this pull request fix one issue?
fix #1737 
fix #1870

### CODE REVIEW

File: config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java [hunk 1]
```diff
@@ -195,6 +198,7 @@ public void run() {
                     setFailResult(configFuture);
                     return;
                 }
+                LOGGER.info("The file name of the operation is {}", fileName);
```
Comment (zjinlei) [inline config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java]: I think it is enough to output once at startup.

Comment (yuhuangbin) [reply config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java]: ok, location has been modified

**Comment by yuhuangbin** [comment]
support exists in where condition
to support issure #1737

**Comment by slievrly** [comment]
@yuhuangbin pr title describes more accurately.

**Pull request merged**